include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -968,7 +1101,7 @@ namespace VectorTools
       mass_matrix.vmult_add(rhs, inhomogeneities);
 
       //now invert the matrix
-      ReductionControl     control(rhs.size(), 0., 1e-12, false, false);
+      ReductionControl     control(5.*rhs.size(), 0., 1e-12, false, false);
```

Comment:
I see. Thanks for the explanation.

@masterleinad -- can you make @kronbichler 's remark into a (slightly shorter) comment that one could put in front of this line and the other, similar ones in this file? -- Thanks!